android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -325,6 +335,30 @@ class DVCClient private constructor(
         }
     }
 
+    private fun refetchConfig(callback: DVCCallback<Map<String, Variable<Any>>>? = null) {
```

Comment:
should your branch now point to main?

Reply:
these changes depend on Elliot's eventsource class so i have to wait for that to go in 

the refetchConfig changes are already in main but i think the DVC-4839-sse-connection branch needs to rebase against main? should be fixed when Elliot's changes are done and everything is in main

Reply:
okay it's on main now